lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -164,7 +164,11 @@ def add_copy_resources_script_phase
             phase = create_or_update_build_phase(native_target, phase_name)
             script_path = target.copy_resources_script_relative_path
             phase.shell_script = %("#{script_path}"\n)
-            # TODO: Add input and output paths for the copy resources script phase.
+            resource_paths_by_config = target.resource_paths_by_config
+            unless resource_paths_by_config.empty?
+              phase.input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
+              phase.output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
```

Comment:
@jgavris how do you tell its re-executing? Does the phase you are running it takes a long time? Note that if you go through the "All" tab in your build log you will see the script but it doesnt show up on "Recent".

Did embed pods frameworks script and check pods manifest lock run again for you also? 

Admittedly the "Copy resources script" is the one that has the highest chance of re-running because of the output folder is an entire folder instead of each individual file unlike "Embed Pods Frameworks" script.

Do you have a sample app for me to test this on?